x/ecocredit/server/marketplace/keeper_test.go

```diff
@@ -75,3 +80,140 @@ func setupBase(t *testing.T) *baseSuite {
 	_, _, s.addr = testdata.KeyTestPubAddr()
 	return s
 }
+
+func (s *baseSuite) createSellOrder(msg *marketplace.MsgSell) []uint64 {
```

Comment:
moved our test utility functions here. i think it might be better to have them all in once place so we can quickly check if something is already implemented or not.